reservations/schema.py

```diff
@@ -141,67 +141,67 @@ def mutate(self, info, **kwargs):
 
         # Send notifications when all m2m relations are saved
-        reservation_saved.send(sender="CreateBerthReservation", reservation=reservation)
+        application_saved.send(sender="CreateBerthApplication", application=application)
 
         ok = True
-        return CreateBerthReservation(berth_reservation=reservation, ok=ok)
+        return CreateBerthApplication(berth_application=application, ok=ok)
 
 
-class CreateWinterStorageReservation(graphene.Mutation):
+class CreateWinterStorageApplication(graphene.Mutation):
     class Arguments:
-        winter_storage_reservation = WinterStorageReservationInput(required=True)
+        winter_storage_application = WinterStorageApplicationInput(required=True)
 
     ok = graphene.Boolean()
-    winter_storage_reservation = graphene.Field(WinterStorageReservationType)
+    winter_storage_application = graphene.Field(WinterStorageApplicationType)
 
     def mutate(self, info, **kwargs):
-        reservation_data = kwargs.pop("winter_storage_reservation")
+        application_data = kwargs.pop("winter_storage_application")
 
-        boat_type_id = reservation_data.pop("boat_type", None)
+        boat_type_id = application_data.pop("boat_type", None)
         if boat_type_id:
-            reservation_data["boat_type"] = BoatType.objects.get(id=int(boat_type_id))
+            application_data["boat_type"] = BoatType.objects.get(id=int(boat_type_id))
 
-        chosen_areas = reservation_data.pop("chosen_areas", [])
+        chosen_areas = application_data.pop("chosen_areas", [])
 
-        reservation = WinterStorageReservation.objects.create(**reservation_data)
+        application = WinterStorageApplication.objects.create(**application_data)
 
         for choice in chosen_areas:
             winter_area_id = from_global_id(choice.get("winter_area_id"))[1]
             winter_storage_area = WinterStorageArea.objects.get(id=winter_area_id)
             WinterStorageAreaChoice.objects.get_or_create(
                 winter_storage_area=winter_storage_area,
                 priority=choice.get("priority"),
-                reservation=reservation,
+                application=application,
             )
 
         # Send notifications when all m2m relations are saved
-        reservation_saved.send(
-            sender="CreateWinterStorageReservation", reservation=reservation
+        application_saved.send(
+            sender="CreateWinterStorageApplication", application=application
         )
 
         ok = True
-        return CreateWinterStorageReservation(
-            winter_storage_reservation=reservation, ok=ok
+        return CreateWinterStorageApplication(
+            winter_storage_application=application, ok=ok
         )
 
 
 class Mutation(graphene.ObjectType):
-    create_berth_reservation = CreateBerthReservation.Field()
-    create_winter_storage_reservation = CreateWinterStorageReservation.Field()
+    create_berth_application = CreateBerthApplication.Field()
```

Comment:
well, these should keep the old names, only internal names can be changed to `Application`

Reply:
Are those the ones being used already?

Reply:
also, the types' names should remain the same from the GQL POV.

Reply:
otherwise frontend will complain about outdated types in their generated typings

Reply:
reservations/tests/test_reservations_graphql.py should pass without any changes 🙂

Reply:
Yes, it's the ones in use.

Reply:
Ok ok. 👍 Reverted to old names.